offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -180,15 +169,32 @@ class OfferToPDFConverter implements OfferExporter {
         List<ProductItem> listOverheadItems = offer.itemsWithOverhead
         List<ProductItem> listNoOverheadItems = offer.itemsWithoutOverhead
 
+        List<ProductItem> listDataGenerationItems = []
+        List<ProductItem> listDataAnalysisItems = []
+        // Sort ProductItems into DataGeneration and Data Analysis
+        listOverheadItems.each {
+            if (it.product instanceof Sequencing) {
+                listDataGenerationItems.add(it)
+            }
+            else {
+                listDataAnalysisItems.add(it)
+            }
+        }
+        //Sort Lists by ProductName
+        listDataAnalysisItems = listDataAnalysisItems.sort{it.product.productName}
+        listNoOverheadItems = listNoOverheadItems.sort{it.product.productName}
+        listDataGenerationItems = listDataGenerationItems.sort{it.product.productName}
+
         //Initialize Number of table
         tableCount = 1
 
         //Initialize Count of ProductItems in table
         tableItemsCount = 1
         int maxTableItems = 8
         //Generate ProductTable for Overhead and Non-Overhead Product Items
-        generateProductTable(OVERHEAD, listOverheadItems, maxTableItems)
-        generateProductTable(NO_OVERHEAD, listNoOverheadItems, maxTableItems)
+        generateProductTable("Data Generation", listDataGenerationItems, maxTableItems)
+        generateProductTable("Data Analysis", listDataAnalysisItems, maxTableItems)
+        generateProductTable("Project Management and Data Storage", listNoOverheadItems, maxTableItems)
```

Comment:
```suggestion
        generateProductTable("Data Generation", listDataGenerationItems, maxTableItems)
        generateProductTable("Data Analysis", listDataAnalysisItems, maxTableItems)
        generateProductTable("Project Management and Data Storage", projectManagementItems + dataStorageItems, maxTableItems)
```
or similar. Since you do not differentiate between with overheads and without overheads anymore?